src/signals/incident/components/form/ContainerSelect/ViewerContainer/ViewerContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+import type { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { ViewerContainer as AscViewerContainer } from '@amsterdam/asc-ui';
+import { MapPanelContext } from '@amsterdam/arm-core';
+
+// Should reflect ViewerContainer props from asc-ui (which are not exported)
+interface AscViewerContainerProps {
```

Comment:
The ViewerContainer is moved to `ARM`. Could you use that one? `arm/packages/arm-core/src/components/ViewerContainer.tsx`

Reply:
I checked that out but it does not explicitly declare the type

Reply:
Please use the ViewerContainer from `arm` the one in `asc-ui` is obsolete.